scripts/lib/rules/NoGet.js

```diff
@@ -5,13 +5,16 @@ const {
 /** @type {import("./rule").Rule} */
 async function validate({ extension, publicEventsFunctions, onError }) {
   // Check that expressions are not prefixed with 'Get'
-  for (const { name, functionType } of publicEventsFunctions) {
+  for (const func of publicEventsFunctions) {
+    const { name, functionType } = func;
     if (
       (functionType === 'Expression' || functionType === 'StringExpression') &&
       name.startsWith('Get') &&
       !legacyGetPrefixedExpressionsExtensions.has(extension.name)
     )
-      onError(`Expression '${name}' is using prohibited 'Get' prefix!`);
+      onError(`Expression '${name}' is using prohibited 'Get' prefix!`, () => {
+        func.name = func.name.replace('Get', '');
```

Comment:
This could break the extension if the expression is used inside the extension? We should probably error and not fix anything.